### DISCUSSION TITLE
How to get rawAdvertisement data on iOS?

### DISCUSSION THREAD

**Discussion by danbaechtold** [capacitor-community/bluetooth-le#507] [Q&A] [answered]
As stated in the docs, when using `requestLEScan()`, on Android the `ScanResult` contains the rawAdvertisement data, but not on iOS.

Is there a reason why, and how else could this data be accessed?

**Comment by pwespi** [comment, marked as answer]
On iOS there is no API to get the raw advertisement data, you can only access the parts separately (as far as I know). But you can get the advertised service and manufacturer data.

**Reply by danbaechtold** [reply]
Thanks for your reply (and awesome work!), @pwespi 
I found an answer which confirms what you say, but states there still is a way.. maybe it's worth checking out? (at least for my use case it would help): https://stackoverflow.com/a/24199063

**Reply by pwespi** [reply]
The post you linked talks about manufacturer data, and this is already implemented (see https://github.com/capacitor-community/bluetooth-le#scanresult). Is this not working for you?

**Reply by danbaechtold** [reply]
Oh, yes of course! This works just fine!
I only had to find the company identifier (which in my case was 65535), and now I can access the data I need.

Thank you so much for your support.
